hiro/windows/window.cpp

```diff
@@ -86,7 +86,11 @@ auto pWindow::frameMargin(s32 width) const -> Geometry {
     SendMessage(hwnd, WM_NCCALCSIZE, (WPARAM)false, (LPARAM)&rcTemp);
     //adjust our previous calculation to compensate for menu wrapping
     rc.bottom += rcTemp.top;
-  }
+  } else {
+    if (bBorderLess){
+	  rc.bottom = 458; // TODO Hardcoded ...
```

Comment:
Why is this hard-coded? What is the intent, why 458px? Even when hardcoded, it would be useful to comment why this value.

Reply:
To prevent the window from sliding up when the menu is also hidden. But it is omitted because there is no switch for the menu to hide the menu.